config/initializers/doorkeeper.rb

```diff
@@ -193,7 +193,7 @@
   #
   after_successful_authorization do |controller|
     # Schedule a cleanup job to clean out over-TTL tokens and grants
-    Delayed::Job.enqueue ::OAuth::CleanupJob.new, priority: ::ApplicationJob.priority_number(:low)
+    ::OAuth::CleanupJob.perform_later
```

Comment:
This needs a low priority which it doesn't have by default. Is it possible to override the priority in the perform call?

Reply:
You would do `Oauth::CleanUpJob.set(priority: :low).perform_later` but in this case in can also be set in the class